contribute.json

```
@@ -2,20 +2,20 @@
   "name": "buildhub2",
   "description": "Mozilla Build Metadata Service",
   "participate": {
-    "irc": "irc://irc.mozilla.org/#storage-team",
-    "irc-contacts": ["peterbe", "mostlygeek"],
+    "irc": "irc://irc.mozilla.org/#breakpad",
+    "irc-contacts": ["willkg"],
```

Let's share the load.
```suggestion
    "irc-contacts": ["willkg", "jwhitlock"],
```

---

Mmm... My inclination is to take it solo for now while you focus on Socorro things and then we can split things up and share more in a month.